pages/dmp_supporting_info/support_01_about_dmp.md

```diff
@@ -34,14 +34,54 @@ This chapter collects administrative information about the DMP itself. It makes
 ## Question-specific guidance
 
 ### DMP title
+Make a title for the data management plan which makes it clear which research project it relates to, if a project has multiple plans, then add information about which part or package of the project this specific plan is for.
+Eg. "Data Management Plan for AMAZING - A really fantastic project, 2020-2024"
+Eg. "Data Management Plan for AMAZING - A really fantastic project, WP3 the complex data collection, 2020-2024"
 
 ### Contributor(s)
+List all person contributing to both the plan and the data management in the project.
+
+Name is written Given-name Sur-name; Eg. Niels Henrik Abel (eller skal det være invertert???)
```

Comment:
I don't think it really matters?

Reply:
om det høstes kan det ha noe å si og det er greit å være konsekvent så det er mulig å håndtere det videre.. men jeg vet ikke hva som er loven - men lettere å få alle til å skrive rett fram enn til å invertere - er jo ingen som skjønner det i DvNO